src/applications/my-education-benefits/config/form.js

```diff
@@ -337,11 +341,16 @@ function renderContactMethodFollowUp(formData, cond) {
   return res;
 }
 
+function transform(metaData, form) {
+  const submission = createSubmissionForm(form.data);
+  return JSON.stringify(submission);
+}
+
 const formConfig = {
   rootUrl: manifest.rootUrl,
   urlPrefix: '/',
-  submit: () =>
-    Promise.resolve({ attributes: { confirmationNumber: '123123123' } }),
+  submitUrl: `${environment.API_URL}/meb_api/v0/submit_claim`,
+  transformForSubmit: transform,
   trackingPrefix: 'my-education-benefits-',
```

Comment:
This config options tells the form system to run the transform option pre-submission.